src/utils/json.ts

```diff
@@ -4,5 +4,6 @@ import { UnprocessableError } from "@root/errors/UnprocessableError"
 
 export const safeJsonParse = Result.fromThrowable(
   JSON.parse,
-  () => new UnprocessableError()
+  () =>
+    new UnprocessableError("The given string could not be parsed into json!")
```

Comment:
Hmm may I get context here? why are we adding this param here ah

Reply:
`message` is a required `string`